### DISCUSSION TITLE
questions around this

### DISCUSSION THREAD

**Discussion by michbsd** [ymgyt/syndicationd#191] [General]
where exactly is the list of my feeds stored?

also, how can I mark an entry as "read" and then filter only for unread entries?

**Comment by ymgyt** [comment]
Thank you for your question.

> where exactly is the list of my feeds stored?

The feeds are stored in a Key Value Store called [kvsd](https://github.com/ymgyt/kvsd). You can find the infrastructure configuration here:
https://github.com/ymgyt/mynix/blob/main/homeserver/modules/kvsd/default.nix

> how can I mark an entry as "read" and then filter only for unread entries?

Unfortunately, there is currently no feature to manage "read" status.

**Comment by michbsd** [comment]
thanks for the answer @ymgyt 
You confirm that the list of subscriptions never leave my local computer? 

Also, concerning read/unread - is this something that will be implemented at some point? It's a pretty basic feature of a RSS reader, and difficult (for me, at least) to live without it.

**Reply by ymgyt** [reply]
> You confirm that the list of subscriptions never leaves my local computer?

Yes, synd does not store the list of subscriptions on the local computer.

> Also, concerning read/unread - is this something that will be implemented at some point?

Yes, managing read/unread status is one of the features we plan to implement eventually.

> It's a pretty basic feature of an RSS reader, and difficult (for me, at least) to live without it.

Thank you for sharing your feedback on essential features for RSS reader users. your input is highly valuable to us.

**Reply by michbsd** [reply]
> Yes, synd does not store the list of subscriptions on the local computer.

That's a mismatched message. Are local feed URLs stored only on the local device, or are they "sent" anywhere?

**Reply by ymgyt** [reply]
> Are local feed URLs stored only on the local device, or are they "sent" anywhere?

They are "sent" somewhere.  
Specifically, they are sent to the `synd-api` hosted by this project.

**Reply by michbsd** [reply]
ah ok.. so using basicauth in feed URL would be **very** bad

**Reply by ymgyt** [reply]
As you mentioned, it’s insecure to include basic auth information in the URL. In the current design, access to each feed is handled on the synd-api side, where the results are cached  and then returned to the client (synd).
